Looks openmvg build failed ,
This is error log, please take a look:
````
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:128 (message):
    Command failed: ninja -v
    Working Directory: D:/buildtrees/openmvg/x86-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      D:\buildtrees\openmvg\config-x86-windows-out.log
````

````
[1/2] "D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -E chdir "D:/buildtrees/openmvg/x86-windows-rel" "D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -S "D:/buildtrees/openmvg/src/1047522df9-4334301758.clean/src" "-DOpenMVG_USE_OPENCV=OFF" "-DOpenMVG_USE_OCVSIFT=OFF" "-DOpenMVG_BUILD_SOFTWARES=ON" "-DOpenMVG_BUILD_GUI_SOFTWARES=ON" "-DOpenMVG_USE_OPENMP=OFF" "-DOpenMVG_BUILD_SHARED=OFF" "-DOpenMVG_BUILD_TESTS=OFF" "-DOpenMVG_BUILD_DOC=OFF" "-DOpenMVG_BUILD_EXAMPLES=OFF" "-DOpenMVG_BUILD_OPENGL_EXAMPLES=OFF" "-DOpenMVG_BUILD_COVERAGE=OFF" "-DOpenMVG_USE_INTERNAL_CLP=OFF" "-DOpenMVG_USE_INTERNAL_COINUTILS=OFF" "-DOpenMVG_USE_INTERNAL_OSI=OFF" "-DOpenMVG_USE_INTERNAL_EIGEN=OFF" "-DOpenMVG_USE_INTERNAL_CEREAL=OFF" "-DOpenMVG_USE_INTERNAL_CERES=OFF" "-DOpenMVG_USE_INTERNAL_FLANN=OFF" "-DOpenMVG_USE_INTERNAL_LEMON=OFF" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" -G "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=D:/packages/openmvg_x86-windows"
FAILED: ../CMakeCache.txt 
"D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -E chdir "D:/buildtrees/openmvg/x86-windows-rel" "D:/downloads/tools/cmake-3.21.1-windows/cmake-3.21.1-windows-i386/bin/cmake.exe" -S "D:/buildtrees/openmvg/src/1047522df9-4334301758.clean/src" "-DOpenMVG_USE_OPENCV=OFF" "-DOpenMVG_USE_OCVSIFT=OFF" "-DOpenMVG_BUILD_SOFTWARES=ON" "-DOpenMVG_BUILD_GUI_SOFTWARES=ON" "-DOpenMVG_USE_OPENMP=OFF" "-DOpenMVG_BUILD_SHARED=OFF" "-DOpenMVG_BUILD_TESTS=OFF" "-DOpenMVG_BUILD_DOC=OFF" "-DOpenMVG_BUILD_EXAMPLES=OFF" "-DOpenMVG_BUILD_OPENGL_EXAMPLES=OFF" "-DOpenMVG_BUILD_COVERAGE=OFF" "-DOpenMVG_USE_INTERNAL_CLP=OFF" "-DOpenMVG_USE_INTERNAL_COINUTILS=OFF" "-DOpenMVG_USE_INTERNAL_OSI=OFF" "-DOpenMVG_USE_INTERNAL_EIGEN=OFF" "-DOpenMVG_USE_INTERNAL_CEREAL=OFF" "-DOpenMVG_USE_INTERNAL_CERES=OFF" "-DOpenMVG_USE_INTERNAL_FLANN=OFF" "-DOpenMVG_USE_INTERNAL_LEMON=OFF" "-DCMAKE_MAKE_PROGRAM=D:/downloads/tools/ninja/1.10.2-windows/ninja.exe" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/a/1/s/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/a/1/s/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/a/1/s" "-D_VCPKG_INSTALLED_DIR=D:/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_DISABLE_SOURCE_CHANGES=ON" -G "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=D:/packages/openmvg_x86-windows"
-- The C compiler identification is MSVC 19.29.30139.0
-- The CXX compiler identification is MSVC 19.29.30139.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- target changed from "" to "auto"
-- Detected CPU: generic
-- Checking for C++11 compiler
-- Checking for C++11 compiler - available
-- Found JPEG: optimized;D:/installed/x86-windows/lib/jpeg.lib;debug;D:/installed/x86-windows/debug/lib/jpeg.lib (found version "62") 
-- Found ZLIB: optimized;D:/installed/x86-windows/lib/zlib.lib;debug;D:/installed/x86-windows/debug/lib/zlibd.lib (found version "1.2.11") 
-- Found PNG: optimized;D:/installed/x86-windows/lib/libpng16.lib;debug;D:/installed/x86-windows/debug/lib/libpng16d.lib (found version "1.6.37") 
-- Found TIFF: optimized;D:/installed/x86-windows/lib/tiff.lib;debug;D:/installed/x86-windows/debug/lib/tiffd.lib (found version "4.3.0")  
CMake Error at CMakeLists.txt:255 (get_target_property):
  get_target_property() called with non-existent target "cereal".
````

--- a/ports/cereal/vcpkg.json
+++ b/ports/cereal/vcpkg.json
@@ -2,5 +2,6 @@
   "name": "cereal",
   "version": "1.3.1",
   "description": "a header-only C++11 serialization library (built in support for binary, XML and JSon)",
-  "homepage": "https://github.com/USCiLab/cereal"
+  "homepage": "https://github.com/USCiLab/cereal",
+  "license": "BSD-3-Clause"
 }

--- a/ports/openmvg/build_fixes.patch
+++ b/ports/openmvg/build_fixes.patch
@@ -99,7 +99,7 @@ index 5d909ec..ab7eb6f 100644
 +endif()
  
 +find_package(cereal REQUIRED CONFIG)
-+get_target_property(CEREAL_INCLUDE_DIRS cereal INTERFACE_INCLUDE_DIRECTORIES)
++get_target_property(CEREAL_INCLUDE_DIRS cereal::cereal INTERFACE_INCLUDE_DIRECTORIES)
 + 
  # ==============================================================================
  # Eigen
@@ -502,7 +502,7 @@ index cc5212f..acc57dd 100644
      $<INSTALL_INTERFACE:include>
  )
 diff --git a/src/openMVG/matching/CMakeLists.txt b/src/openMVG/matching/CMakeLists.txt
-index 2e0eafd..be0006f 100644
+index 2e0eafd7..d54d1e88 100644
 --- a/src/openMVG/matching/CMakeLists.txt
 +++ b/src/openMVG/matching/CMakeLists.txt
 @@ -16,6 +16,7 @@ list(REMOVE_ITEM matching_files_cpp ${REMOVEFILESUNITTEST})
@@ -513,9 +513,12 @@ index 2e0eafd..be0006f 100644
  
  set_source_files_properties(${matching_files_cpp} PROPERTIES LANGUAGE CXX)
  add_library(openMVG_matching
-@@ -37,10 +38,10 @@ target_link_libraries(openMVG_matching
+@@ -35,12 +36,12 @@ target_link_libraries(openMVG_matching
+   PUBLIC
+     openMVG_features
      Threads::Threads
-     cereal
+-    cereal
++    cereal::cereal
  )
 -if (NOT DEFINED OpenMVG_USE_INTERNAL_FLANN)
 +if (NOT OpenMVG_USE_INTERNAL_FLANN)
@@ -706,3 +709,55 @@ index 0d61c9f..05c8b29 100644
    #Configure Eigen install
    set(INCLUDE_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/include/openMVG/third_party/eigen)
    add_subdirectory(eigen)
+diff --git a/src/openMVG/cameras/CMakeLists.txt b/src/openMVG/cameras/CMakeLists.txt
+index 333ab729..50c8f572 100644
+--- a/src/openMVG/cameras/CMakeLists.txt
++++ b/src/openMVG/cameras/CMakeLists.txt
+@@ -4,7 +4,7 @@ set_property(TARGET openMVG_camera APPEND PROPERTY
+   INTERFACE_INCLUDE_DIRECTORIES "$<INSTALL_INTERFACE:include>")
+ 
+ target_compile_features(openMVG_camera INTERFACE ${CXX11_FEATURES})
+-target_link_libraries(openMVG_camera INTERFACE openMVG_numeric cereal ${OPENMVG_LIBRARY_DEPENDENCIES})
++target_link_libraries(openMVG_camera INTERFACE openMVG_numeric cereal::cereal ${OPENMVG_LIBRARY_DEPENDENCIES})
+ install(TARGETS openMVG_camera DESTINATION lib EXPORT openMVG-targets)
+ 
+ UNIT_TEST(openMVG Camera_Pinhole openMVG_camera)
+diff --git a/src/openMVG/features/CMakeLists.txt b/src/openMVG/features/CMakeLists.txt
+index 2e49701d..0b5e1c3d 100644
+--- a/src/openMVG/features/CMakeLists.txt
++++ b/src/openMVG/features/CMakeLists.txt
+@@ -31,7 +31,7 @@ target_include_directories(openMVG_features
+ )
+ target_link_libraries(openMVG_features
+   PRIVATE openMVG_fast ${STLPLUS_LIBRARY}
+-  PUBLIC ${OPENMVG_LIBRARY_DEPENDENCIES} cereal)
++  PUBLIC ${OPENMVG_LIBRARY_DEPENDENCIES} cereal::cereal)
+ if (MSVC)
+   set_target_properties(openMVG_features PROPERTIES COMPILE_FLAGS "/bigobj")
+   target_compile_options(openMVG_features PUBLIC "-D_USE_MATH_DEFINES")
+diff --git a/src/openMVG/geometry/CMakeLists.txt b/src/openMVG/geometry/CMakeLists.txt
+index 0a6a016a..e509a4d5 100644
+--- a/src/openMVG/geometry/CMakeLists.txt
++++ b/src/openMVG/geometry/CMakeLists.txt
+@@ -21,7 +21,7 @@ set_property(TARGET openMVG_geometry PROPERTY FOLDER OpenMVG/OpenMVG)
+ target_link_libraries(openMVG_geometry
+   PUBLIC
+     openMVG_numeric
+-    cereal
++    cereal::cereal
+   PRIVATE
+     openMVG_linearProgramming
+ )
+diff --git a/src/openMVG/sfm/CMakeLists.txt b/src/openMVG/sfm/CMakeLists.txt
+index b8928083..1d10f939 100644
+--- a/src/openMVG/sfm/CMakeLists.txt
++++ b/src/openMVG/sfm/CMakeLists.txt
+@@ -25,7 +25,7 @@ target_link_libraries(openMVG_sfm
+     openMVG_graph
+     openMVG_matching
+     openMVG_multiview
+-    cereal
++    cereal::cereal
+     ${OPENMVG_LIBRARY_DEPENDENCIES}
+ )
+ 

--- a/ports/openmvg/vcpkg.json
+++ b/ports/openmvg/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "openmvg",
   "version": "2.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "open Multiple View Geometry library. Basis for 3D computer vision and Structure from Motion.",
   "dependencies": [
     "cereal",

--- a/ports/openmvg/vcpkg.json
+++ b/ports/openmvg/vcpkg.json
@@ -3,6 +3,7 @@
   "version": "2.0",
   "port-version": 2,
   "description": "open Multiple View Geometry library. Basis for 3D computer vision and Structure from Motion.",
+  "license": "MPL-2.0-no-copyleft-exception",
   "dependencies": [
     "cereal",
     {

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5054,7 +5054,7 @@
     },
     "openmvg": {
       "baseline": "2.0",
-      "port-version": 1
+      "port-version": 2
     },
     "openmvs": {
       "baseline": "1.1.1",

--- a/versions/c-/cereal.json
+++ b/versions/c-/cereal.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "18d0d653f7935314e271ea0d1d0d3b4256310b39",
+      "git-tree": "e292069bec2c118568ea23a3db965037a9e38790",
       "version": "1.3.1",
       "port-version": 0
     },

--- a/versions/o-/openmvg.json
+++ b/versions/o-/openmvg.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "65b55dbb8a7097c654ec57a840c1971c3b87b574",
+      "version": "2.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "eea9187e35c87676d442453e6942a47417d7c2dd",
       "version": "2.0",
       "port-version": 1

--- a/versions/o-/openmvg.json
+++ b/versions/o-/openmvg.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "65b55dbb8a7097c654ec57a840c1971c3b87b574",
+      "git-tree": "894793611702efbc0abc44768002cdd12f73557a",
       "version": "2.0",
       "port-version": 2
     },

